### PR TITLE
BZ#2016983 Tweak install dir param syntax to avoid syntax issue with tilda

### PR DESCRIPTION
**Pull request by codyhoag**
https://bugzilla.redhat.com/show_bug.cgi?id=2016983

Fixed across many modules. Example preview can be viewed here: https://deploy-preview-39453--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-customizations.html#installation-initializing_installing-azure-customizations

### CODE REVIEW

**Comment by codyhoag** [comment]
@MayXuQQ can you verify? Thanks!

**Review by MayXuQQ — APPROVED** [review]

File: modules/installation-getting-debug-information.adoc [hunk 1]
```diff
@@ -19,6 +19,6 @@ $ cat ~/<installation_directory>/.openshift_install.log <1>
 +
 [source,terminal]
 ----
-$ ./openshift-install create cluster --dir=<installation_directory> --log-level=debug <1>
+$ ./openshift-install create cluster --dir <installation_directory> --log-level=debug <1>
```
Comment (MayXuQQ) [inline modules/installation-getting-debug-information.adoc:22]: ./openshift-install create cluster --dir <installation_directory> --log-level debug

File: modules/installation-uninstall-clouds.adoc [hunk 2]
```diff
@@ -49,7 +49,7 @@ cluster.
 [source,terminal]
 ----
 $ ./openshift-install destroy cluster \
---dir=<installation_directory> --log-level=info <1> <2>
+--dir <installation_directory> --log-level=info <1> <2>
```
Comment (MayXuQQ) [inline modules/installation-uninstall-clouds.adoc:52]: --dir <installation_directory> --log-level info <1> <2>

File: modules/specifying-openshift-installer-log-levels.adoc [hunk 3]
```diff
@@ -17,6 +17,6 @@ By default, the {product-title} installer log level is set to `info`. If more de
 +
 [source,terminal]
 ----
-$ ./openshift-install --dir=<installation_directory> wait-for bootstrap-complete --log-level=debug  <1>
+$ ./openshift-install --dir <installation_directory> wait-for bootstrap-complete --log-level=debug  <1>
```
Comment (MayXuQQ) [inline modules/specifying-openshift-installer-log-levels.adoc:20]: $ ./openshift-install --dir <installation_directory> wait-for bootstrap-complete --log-level debug  <1>

**Comment by mjpytlak** [comment]
Peer review /lgtm.

**Pull request merged**